### PR TITLE
Add v0.6.0 release note.

### PR DESCRIPTION
**Pull request by nilo19**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

Add v0.6.0 release note.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
Add v0.6.0 release note.
```

### CODE REVIEW

**Comment by nilo19** [comment]
need to build ccm and cnm image and push to mcr

File: site/content/en/blog/releases/v0.6.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+## Major changes since v0.5.0
+
+-------
+
+- Increase the e2e test coverage for cluster autoscaler([#364](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/364))
+- Use hugo to generate doc website([#358](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/358))
+- Update E2E test related docs and script([#355](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/355))
+- Update vendor against release 1.18([#354](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/354))
```
Comment (feiskyer) [inline site/content/en/blog/releases/v0.6.0.md:16]: this should be 1.19

Comment (feiskyer) [reply site/content/en/blog/releases/v0.6.0.md:16]: release-1.18 updates is part of v0.5.1

Comment (nilo19) [reply site/content/en/blog/releases/v0.6.0.md:16]: we have not done such an operation to update vendor against release 1.19.

Comment (nilo19) [reply site/content/en/blog/releases/v0.6.0.md:16]: will update it now

**Comment by nilo19** [comment]
test blocked by `Reason:ContainersNotReady`

**Pull request merged**